src/main/java/com/Model/Event.java

```diff
@@ -35,9 +35,10 @@ public Event(){
         this.place = new Place();
         Calendar calendar = Calendar.getInstance();
         calendar.set(2000, 01, 01);
-        this.date = new Date (calendar.getTimeInMillis()); //By default set date to 1st/Jan/2000
+        this.date = calendar.getTime(); //By default set date to 1st/Jan/2000
```

Comment:
the default constructor should call the other constructor (this avoids code duplication).